<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmaking</class>
 <widget class="QDialog" name="cmaking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	background-color: rgb(255, 255, 255);
}

QFrame#frame{
	border-radius: 30px;
	border-image: url(:/res/goodBKG.jpg);
}

#frame_2{
	border-radius: 30px;
	background-color: rgba(158, 158, 158, 120);
}

QLabel#label{
	border-radius: 30px;
	background-color: rgba(98, 98, 98, 120);
}

QLabel{
	background: transparent;
	color: rgb(255, 255, 255);
	font: 10pt &quot;等线&quot;;
}

QLabel#label_2{
	background: transparent;
	color: rgb(255, 255, 255);
	font: 20pt &quot;等线&quot;;
}

QPushButton{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 14pt &quot;等线&quot;;
}

QPushButton:hover{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 14pt &quot;等线&quot;;
}

QPushButton:pressed{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 14pt &quot;等线&quot;;
}

QPushButton#pushButton_3{
	border-radius: 10px;
	background: transparent;
	font: 20pt &quot;等线&quot;;
}

QLineEdit{
	background: transparent;
	border: none;
	border-bottom: 1px solid rgba(255, 255, 255, 180);
	color: rgba(255,255,255,200);
	font: 12pt &quot;等线&quot;;
selection-background-color: green;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>384</width>
     <height>438</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>384</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>384</width>
     <height>438</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>384</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>384</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_back">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>201</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_check_answer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请再输入一遍您的密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="answer_len_warning">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>密码长度不足六位！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="answer_check_warning">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>两次输入的密码不相同！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="success">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>注册成功！请转至登录！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_done">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>请输入您的账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_turn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>转至登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="id_same_warning">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>该账号已经注册过了！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin_answer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>输入大于六位且无汉字的密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
